Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -6004,6 +6004,10 @@ local Hash = cCryptoHash.sha1HexString("DataToHash")
 			},
 			Constants =
 			{
+				INVALID_ID =
+				{
+					Notes = "Special entity id, that is used for functions that returns a zero. For example if the spawning with cWorld:SpawnMob failed.",
```

Comment:
`Special value of an entity ID, indicating a failure. Used primarily in functions that create entities when the entity cannot be created.`